src/test/java/org/opensearch/neuralsearch/query/aggregation/BucketAggregationsWithHybridQueryIT.java

```diff
@@ -68,165 +68,165 @@ public class BucketAggregationsWithHybridQueryIT extends BaseAggregationsWithHyb
 
     @SneakyThrows
     public void testBucketAndNestedAggs_whenAdjacencyMatrix_thenSuccessful() {
-        updateClusterSettings(CLUSTER_SETTING_CONCURRENT_SEGMENT_SEARCH, false);
```

Comment:
Do not review this class as it already merged in main. The change is showing here because we have cherry picked the commit.